package.json

```diff
@@ -18,14 +18,18 @@
     "test": "gulp test",
     "test:watch": "gulp test-watch"
   },
+  "peerDependencies": {
+    "backbone": ">= 1.1.2",
+    "underscore": ">= 1.4.4"
+  },
   "dependencies": {
-    "backbone": "1.1.2",
+    "backbone": "1.0.0 - 1.1.2",
```

Comment:
We should remove this. We don't this this library to pull in its own copy of backbone if the host has the package. Right?

Reply:
Let's move them to `devDependencies`